lighthouse-core/scripts/json-size.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview A script for measuring the size of an each property of an object.
+ * Primary purpose is to better understand what takes up the most space in an LHR.
+ */
+
+/* eslint-disable no-console */
+
+// cat lhr.json | node lighthouse-core/scripts/json-size.js | less
+// cat lhr.json | jq .audits | node lighthouse-core/scripts/json-size.js | less
+
+const fs = require('fs');
+const inputJson = fs.readFileSync(0, 'utf-8');
```

Comment:
holy crap srsly?

i clearly read the wrong SO answer because i was using this like a pleb https://stackoverflow.com/a/20087094/89484